### ISSUE TITLE
edomi visu not starting... always red rotating circle

### ISSUE THREAD

**Issue by Civlo85** [starwarsfan/edomi-docker#5]
Hi,

My Problem is, that I am able to create visualizations, add buttons, stuff here and there. Everything seems to work fine. But when I try to start the visu, I always get the red rotating circle. Does not matter if the visu is empty (just a text) or full with more complex stuff.
![image](https://user-images.githubusercontent.com/70633987/200668894-410e3a53-56b1-4c31-9b67-4012660a01ac.png)

My setup:
Running edomi in a docker container.

Image: starwarsfan/edomi-docker:2.03.5	

Ports:
22222:22/tcp
3671:3671/udp
50000:50000/udp
50001:50001/udp
81:80/tcp
8080:8080/tcp

Envs:
WEBSOCKETPORT=8080
HOSTIP=192.168.178.253

Restart: On failure

Created and mapped volumes to:
/usr/local/edomi
/var/edomi-backups
/var/lib/mysql

I call the visu with the following url:

http://192.168.178.253:81/visu/

Then I got asked to enter created user credentials:
![image](https://user-images.githubusercontent.com/70633987/200671286-6e00f658-47f7-4723-9277-4cacddd8b435.png)

After clicking "OK" I see the red circle

Any ideas here?

Thank you very much

**Comment by Civlo85** [comment]
Additional infos:

it has something to do with the web socket. But I don't know, how to solve it.

Chrome console:
![image](https://user-images.githubusercontent.com/70633987/200675097-38855078-4c6d-4ac2-bd02-978dd37b9aed.png)

Both, docker host and docker container (= connected via ssh to edomi) are listening on port 8080
![image](https://user-images.githubusercontent.com/70633987/200675373-4f1d7045-dc87-4167-8190-50f0bd2616fe.png)

**Comment by Civlo85** [comment]
Ok, next information:
![image](https://user-images.githubusercontent.com/70633987/200678068-90639446-9012-437d-b8c4-f74ffe117c81.png)

If you look closely, the host of this port listening differs. 127.0.0.1:8080 means, that this container is only accepting connections from same machine. But this is not the case. 

When I try to modify the setting in the edomi.ini, the changes are written back to "127.0.0.1" after each restart of the container. So where is the data coming from?
![image](https://user-images.githubusercontent.com/70633987/200678457-0b0ba17c-749e-4a77-a692-6f97badff8cf.png)

**Comment by Civlo85** [comment]
Tada. Solved 😄
![image](https://user-images.githubusercontent.com/70633987/200680933-fbc42dc4-b278-4b05-b7e7-a5c49336f3bf.png)

The error in my case was in the file /root/start.sh in the docker container. See here:
![image](https://user-images.githubusercontent.com/70633987/200680268-e55bc97f-8346-4109-9d8d-1de682e27b33.png)

This line overwrites the visu ip permanently to 127.0.0.1 
So I changed it to the following:
![image](https://user-images.githubusercontent.com/70633987/200681324-23d72dc5-d604-4aa0-bfc3-a005015bb8f0.png)

and after that, the visu is working! And looking at the port binding:
![image](https://user-images.githubusercontent.com/70633987/200681131-edaefb49-2a43-4e87-8c50-0791f4a4705e.png)

Now it is listening to connections from outside.

I can't imagine that I am the only one with that problem?! Found nothing in the support forum.
